sdk/python/v1beta1/kubeflow/katib/api/katib_client.py

```diff
@@ -251,8 +252,8 @@ def list_experiments(self, namespace=None):
         :param namespace: Experiments namespace.
         If the namespace is None, it takes "default" namespace.
 
-        :return: List of Experiment names with the statuses.
-        :rtype: list[dict]
+        :return: List of Experiment objects.
+        :rtype: list[V1beta1Experiment]
```

Comment:
Please could you also modify this doc: https://github.com/kubeflow/katib/blob/master/sdk/python/v1beta1/docs/KatibClient.md#list_experiments.

Reply:
Yeap, I should've done that.
By the way, I guess the `make generate` doesn't work for generating docs/KatibClient.md since `hack/gen-python-sdk/post_gen.py` doesn't handle /docs/KatibClient.md. Is it intended ?

Reply:
Unfortunately, we don't have script to automatically generate `KatibClient.md` doc, currently we manually update it. We might need to modify `post_gen.py` in the future.

Reply:
DONE! Thanks for the explanation!